cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -132,6 +132,7 @@ def _add_login_nodes_pool_launch_template(self):
                                 "CustomAwsBatchCliPackage": self._config.custom_aws_batch_cli_package or "",
                                 "CWLoggingEnabled": "true" if self._config.is_cw_logging_enabled else "false",
                                 "DirectoryServiceEnabled": str(ds_config is not None).lower(),
+                                "DirectoryServiceReadOnlyUser": ds_config.domain_read_only_user,
```

Comment:
What if DirectorySerbvice configuration is missing? We should check for ds_config being None, right?

Reply:
Yes!
Good catch !
A neuron in my brain thought about this but it was suppressed ! :D